recipes/unittest2/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "unittest2" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "22882a0e418c284e1f718a822b3b022944d53d2d908e1690b319a9d3eb2c0579" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - no-argparse-req.patch
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python <3
```

Comment:
Technically it does work on Python 3. Not sure if we want to worry about that though.

Reply:
The PyPI page says that it is "tested to run on" Python versions only up to 3.4 ... I would imagine that that doesn't preclude compatibility with the higher versions that we use, but <shrug>.

Reply:
Yeah, let's not worry about it. We shouldn't really encourage people to use a backport in Python versions where standard options exist.

Reply:
Yeah, that was also part of my thinking.